libs/sdk-backend-spi/src/workspace/execution/results.ts

```diff
@@ -1,27 +1,86 @@
-// (C) 2019-2020 GoodData Corporation
+// (C) 2019-2022 GoodData Corporation
 
 import { ObjRef } from "@gooddata/sdk-model";
 import isEmpty from "lodash/isEmpty";
 
 /**
- * Single calculated data value. The data value may be `null` - the semantics here are same as with
+ * Single calculated data value.
+ *
+ * @remarks
+ * The data value may be `null` - the semantics here are same as with
  * SQL nulls. The calculated numeric value WILL be returned in string representation - this is to
  * prevent float number precision errors.
  *
  * @public
  */
 export type DataValue = null | string | number;
 
+/**
+ * Descriptor of the measure and its contents.
+ *
+ * @public
+ */
+export interface IMeasureDescriptorObject {
+    items: IMeasureDescriptor[];
+    totalItems?: ITotalDescriptor[];
+}
+
 /**
  * Describes measure group and its contents.
  * @public
  */
 export interface IMeasureGroupDescriptor {
     // TODO: rename this to measureGroupDescriptor ... the goal is to get rid of the overused 'header' nomenclature
-    measureGroupHeader: {
-        items: IMeasureDescriptor[];
-        totalItems?: ITotalDescriptor[];
-    };
+    measureGroupHeader: IMeasureDescriptorObject;
+}
+
+/**
+ * Measure descriptor object.
+ *
+ * @public
+ */
+export interface IMeasureHeaderItem {
```

Comment:
Should be `IMeasureDescriptorItem`, we are trying to get rid of the overused header stuff.